java/sidecar/src/main/java/com/instaclustr/cassandra/sidecar/JMXHelper.java

```diff
@@ -0,0 +1,23 @@
+package com.instaclustr.cassandra.sidecar;
+
+import javax.management.MBeanServerConnection;
+import javax.management.remote.JMXConnector;
+import javax.management.remote.JMXConnectorFactory;
+import javax.management.remote.JMXServiceURL;
+
+import java.io.IOException;
+
+import com.instaclustr.picocli.typeconverter.JMXServiceURLTypeConverter;
+
+public class JMXHelper {
+
+    public MBeanServerConnection getMBeanServerConnection(String jmxServiceURL) throws IOException {
```

Comment:
these can be `static` (to avoid having to call `new JMXHelper`)

Reply:
I do not want to have this necessarily static as static stuff is notoriously known for being harder to mock. In this particular case we should provide JMXHelper to `Sidecar` class on instantiation from outside.